code/ZAS/Airflow.dm

```diff
@@ -78,7 +79,7 @@ atom/movable/GotoAirflowDest(n)
 
 	if(knockdown <= 0)
 		to_chat(src, "<span class='warning'>The sudden rush of air knocks you over!</span>")
-	SetKnockdown(rand(1,5))
```

Comment:
Jesus fucking christ it was random all along?

Reply:
😂